### PR TITLE
tr(groovy scripts): document the new security Groovy scripts mechanism

### PR DESCRIPTION
**Pull request by educhastenier**
Strongly encourage the use of default package

Closes [BS-17237](https://bonitasoft.atlassian.net/browse/BS-17237)

### CODE REVIEW

File: md/rest-api-authorization.md [hunk 1]
```diff
@@ -125,26 +125,46 @@ This enables you to tailor the permissions needed to access a resource using dyn
...
+    #DELETE|bpm/case=[profile|Administrator, check|org.bonitasoft.permissions.CasePermissionRule]
+    #GET|bpm/archivedCase=[profile|Administrator, check|org.bonitasoft.permissions.CasePermissionRule]
 ```
 
 To specify a dynamic check for a method and resource, uncomment the line in the file `dynamic-permissions-checks-custom.properties` and add the conditions.
-If you specify a condition that calls a Groovy script, add the script to the `tenant_security_scripts` folder. Then use the [plaform setup tool](BonitaBPM_platform_setup.md) to save the changes.
+If you specify a condition that calls a Groovy script, you must add the new script:
+
+If the platform has never been started yet:
+* add the script to the `setup/platform_conf/current/tenant_template_security_scripts` folder
```
Comment (laurentleseigneur) [inline md/rest-api-authorization.md]: for first run, it should be the `initial` folder

**Review by laurentleseigneur — CHANGES_REQUESTED** [review]

File: md/rest-api-authorization.md [hunk 2]
```diff
@@ -125,26 +125,46 @@ This enables you to tailor the permissions needed to access a resource using dyn
...
+
+The `tenant_security_scripts` folder contains a script sample that can be used to write your own.
+Bonita also provides default scripts that should fit common usages. They are packages internally in the binaries, but the
+[source code is available](https://github.com/bonitasoft/bonita-engine/tree/master/bpm/bonita-core/bonita-process-engine/src/main/groovy/org/bonitasoft/permissions).
+These provided scripts can be used as a base for you own scripts.
+
+If you write your own scripts:
+* make sure you either inherit from an existing rule, or implement the PermissionRule interface, by overriding the isAllowed() method
+* make sure you use the default package declaration at the top of your groovy class (no `package` keyword used)
+* make sure this .groovy file is placed in the default directory, under 'initial/tenant_template_security_scripts/' or under 'current/tenants/TENANT_ID/tenant_security_scripts/'
```
Comment (abirembaut) [inline md/rest-api-authorization.md]: I would just add "if the platform has already been started" at the end of this sentence

**Review by abirembaut — COMMENTED** [review]
LGTM
Just a small comment

File: md/rest-api-authorization.md (hunk 1 above)
Comment (educhastenier) [reply md/rest-api-authorization.md]: nice catch

**Review by laurentleseigneur — APPROVED** [review]

**Pull request merged**